README.md

```diff
@@ -522,6 +527,10 @@ Other Style Guides
     function getFullName({ firstName, lastName }) {
       return `${firstName} ${lastName}`;
     }
+    
+    // best - 2
+    const  getFullName = ({ firstName, lastName }) => `${firstName} ${lastName}`;
+    
```

Comment:
using an arrow function isn't equally as good as using a normal named function; and in this case I'd say it's worse, because you're relying on implicit function name inference.